specification/configuration.md

```diff
@@ -130,6 +136,32 @@ beyond the OpenTelemetry specification exist.
   - Distribution MUST default to `otlp`
   - Distribution MUST offer `jaeger-thrift-splunk` that defaults to `http://127.0.0.1:9080/v1/trace`
 
+### Real User Monitoring Libraries
+
+Real User Monitoring (RUM) instrumentation libraries cannot use environment variables for configuration.
+Instead, they MUST expose a `SplunkRum` object/class/namespace (depending on the language used) that allows
+setting the following properties (in a language-specific way: Java may use builders, Swift can use named
+parameters with default values, JavaScript can use objects, etc.):
+
+| Configuration property (default value) | Description                                                                                                                                                                                                 | Required |
+| -------------------------------------- | -----------                                                                                                                                                                                                 | -------- |
+| `realm` ()                             | Splunk realm, e.g. `us0`, `us1`. If set, value of `beaconUrl` will be automatically computed based on this. [1] [2]                                                                                         | Yes      |
+| `beaconUrl` ()                         | RUM beacon URL, ex. `https://rum-ingest.<realm>.signalfx.com/v1/rum`. If both `realm` and `beaconUrl` are set, `beaconUrl` takes precedence. [1] [2]                                                        | Yes      |
+| `rumAuth` ()                           | RUM authentication token. [1]                                                                                                                                                                               | Yes      |
+| `applicationName` ()                   | Instrumented application name. [1]                                                                                                                                                                          | Yes      |
```

Comment:
Continuing on the consistency journey -- we use "service name".

```suggestion
| `serviceName` ()                   | Instrumented service name.                                                                                                                                                                          | Yes  [1]    |
```

Reply:
That's an interesting topic to discuss. Typically "service" is used in a server context. RUM however is client-side. Calling this "service" instead of "application" (which is context-agnostic) may cause some misunderstanding (ie "is this name for a service that ingests the RUM data?").

Reply:
I think I agree with that, although calling a mobile or a web app a "service" is a bit iffy; also see https://github.com/signalfx/gdi-specification/pull/112#discussion_r657314903

Reply:
Naming is hard -- we technially call it application instrumentation versus serverless instrumentation today -- these refer to server-side. I am not sure if the brower/client is an app -- clearly that term is used on mobile devices.